commands/weatherCommands.mjs

```diff
@@ -1,19 +1,21 @@
 /* eslint-disable import/extensions */
+/* eslint-disable no-console */
+/* eslint-disable arrow-body-style */
 /**
  * @fileoverview
  * Bot commands for interacting with the Dark Sky API for weather information
  */
-import simplesky from 'simplesky';
-import { sendMessageWithOptions } from '../utils/Utils.mjs';
+import Simplesky from 'simplesky';
```

Comment:
Have you verified this work? I know when I originally wrote it it was very particular about me using lowercase

Reply:
The import does work, yes.

Reply:
No, does the actual weather command still function with this?

Reply:
The command still functions normally. I just tested it.